lib/customs.js

```diff
@@ -14,6 +14,11 @@ module.exports = function (log, error) {
 
   // Perform a deep clone of payload and remove user password.
   function sanitizePayload(payload) {
+
+    if (! payload) {
+      return
```

Comment:
When would `payload` be undefined? Is this something hapi does for endpoints or is this for tests?